docs/source/conceptual/tensor_parallelism.md

```diff
@@ -0,0 +1,15 @@
+# Tensor Parallelism
+
+Tensor parallelism (also called horizontal model parallelism) is a technique used to fit a large model in multiple GPUs.  Intermediate outputs between ranks are sent and received from one rank to another in a synchronous or asynchronous manner. When multiplying input with weights for inference, multiplying input with weights directly is equivalent to dividing the weight matrix column-wise, multiplying each column with input separately, and then concatenating the separate outputs like below 👇 
+
+![Image courtesy of Anton Lozkhov](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/tgi/TP.png)
+
+In TGI, tensor parallelism is implemented under the hood by sharding weights and placing them in different ranks. The matrix multiplications then take place in different ranks and are then gathered into a single tensor. 
+
+<Tip warning={true}>
+
+Tensor Parallelism only works for models officially supported, it will not work when falling back on `transformers`.
```

Comment:
Maybe link to the part that explains custom models once that's merged

Reply:
This is unresolved I think

Reply:
It's not merged yet

Reply:
I addressed this nevertheless.